ha/setup/create_pacemaker_resources.py

```diff
@@ -118,11 +123,19 @@ def motr(cib_xml, push=False, **kwargs):
                     not_defined motr-confd-count or motr-confd-count lt integer {quorum_size}")
             process.run_cmd(f"pcs -f {cib_xml} constraint order hax-clone then motr-ios-{i}-clone")
             process.run_cmd(f"pcs -f {cib_xml} constraint colocation add motr-ios-{i}-clone with hax-clone")
+            process.run_cmd(f"pcs -f {cib_xml} constraint order stop motr-ios-{i}-clone then stop motr-confd-1-clone")
         except Exception as e:
             raise CreateResourceConfigError(f"Invalid node_count. Error: {e}")
     if push:
         cib_push(cib_xml)
 
+def stop_constraint_on_motr_ios(resource_name, cib_xml, push=False, **kwargs):
```

Comment:
should we increase stop timeout to 6min 
motr service have current time stop in systemd is 5 min

so seen many time stop is failing

Reply:
Lets observe the system after these changes. If required more timeout, we will ask motr team to change it. By the way, the motr-ios timeout in pacemaker is 120 sec.